protos/xyz/block/ftl/v1/ftl.proto

```diff
@@ -265,3 +265,38 @@ service ObservabilityService {
 
   rpc SendMetric(SendMetricRequest) returns (SendMetricResponse);
 }
+
+message MetricCounter {
+  int64 value = 1;
+}
+
+message MetricHistorgram {
+  int64 count = 1;
+  int64 sum = 2;
+  repeated uint64 bucket = 3;
+}
+
+message Verb {
```

Comment:
This is too generic an identifier to use for the top level, as it's specific to the UI.

Maybe we should split the ConsoleService out into its own .proto file/package?

Reply:
Yeah I like that idea. I think it will make it easier for FE development of the console too to have a dedicated playground.